src/Quizzer.API.DAL/UnitOfWork/IUnitOfWork.cs

```diff
@@ -11,10 +13,25 @@ public interface IUnitOfWork : IDisposable {
         /// <returns></returns>
         IRepository<TEntity> GetRepository<TEntity>()
             where TEntity : class, IEntity;
-           
+
         /// <summary>
         /// Makes changes on entities in database
         /// </summary>
         void Commit();
+
+        /// <summary>
+        /// Discards changes
+        /// </summary>
+        public void Dispose();
```

Comment:
This method hides the Dispose() member from IDisposable, remove it.

Reply:
removed.